kieserver/image.yaml

```diff
@@ -142,6 +142,30 @@ envs:
   - name: "KIE_SERVER_DISABLE_KC_VERIFICATION"
     example: "true"
     description: "Disable Maven KIE Jar verification. It is recommended to test the kjar manually before disabling this verification."
+  - name: "org.kie.kafka.server.ext.disabled"
+    example: "true"
+    description: "Disable Kafka extension"
+  - name: "org.kie.server.jbpm-kafka.ext.topics"
+    example: "event=events,book=books"
+    description: "message/signal name>=<topicName> for every topic that needs to be mapped globally, by default topic name will be equal to message/signal name"
+  - name: "org.kie.server.jbpm-kafka.ext.bootstrap.servers"
+    example: "localhost:9092"
+    description: " A comma separated list of host/port pairs to use for establishing the initial connection to the Kafka cluster"
+  - name: "org.kie.server.jbpm-kafka.ext.client.id"
+    example: "applicationName"
+    description: "An id string to pass to the server when making requests. Not passed by default (the purpose of this property is to provide a logical application name for logging)"
+  - name: "org.kie.server.jbpm-kafka.ext.allow.auto.create.topics"
+    example: "true"
+    description: "Allow automatic topic creation.Allow automatic topic creation. Enabled by default"
+  - name: "org.kie.server.jbpm-kafka.ext.group.id"
+    example: "jbpm-consumer"
+    description: "A unique string that identifies the group this consumer belongs to. Default to 'jbpm-consumer'"
+  - name: "org.kie.server.jbpm-kafka.ext.acks"
+    example: "1"
+    description: "The number of acknowledgments the producer requires the leader to have received before considering a request complete. 1 by default, which means the leader will write the record to its local log but will respond without awaiting full acknowledgement from all followers"
+  - name: "org.kie.server.jbpm-kafka.ext.max.block.ms"
```

Comment:
shouldn't the env name be here and the system property on ks image?

Reply:
Updated